### PR TITLE
fixed typos and removed unused code

### PR DESCRIPTION
**Pull request by carlosp420**
Found more typos and removed variables that are declared, computed but not used in the code. Also removed a section of conditionals that compute a variable that is not used/returned in the function.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/codonalign/codonseq.py [hunk 1]
```diff
@@ -210,7 +210,7 @@ def translate(
                 )
         return "".join(amino_acids)
 
-    def toSeq(self, alphabet=generic_dna):
```
Comment (carlosp420) [inline Bio/codonalign/codonseq.py:213]: the only usages of this method in Biopython are two instances of:

```
        alignments = [SeqRecord(rec.seq.toSeq(), id=rec.id) for rec in self._records]
```

However this method is not private and maybe breaks users functionality? unsure.

Comment (peterjc) [reply Bio/codonalign/codonseq.py:213]: We're dropping alphabet usage as part of the next release. While on very commonly used function signatures (like ``Bio.SeqIO.parse``) I think we need to consider something to ease the transition, here though removal seems OK. @mdehoon do you agree?

**Comment by peterjc** [comment]
I think we need @zruan to comment on the proposed code removal in ``Bio/codonalign/codonseq.py`` - was this internal variables not actually used in the methods' output?

**Comment by peterjc** [comment]
Any idea why Sphinx should be failing to build the API docs?

```
Handler <function mangle_signature at 0x7f8ae571b620> for event 'autodoc-process-signature' threw an exception
Makefile:20: recipe for target 'html' failed
```

File: Bio/codonalign/codonseq.py (hunk 1 above)
Comment (mdehoon) [reply Bio/codonalign/codonseq.py:213]: Yes let's remove it.

**Comment by carlosp420** [comment]
@peterjc all tests passed now after another git push. Yes the remove variables were declared internally but actually not used to compute the output.

**Comment by peterjc** [comment]
Thank you!

**Pull request merged**